cloud/kubernetes/helm/templates/db-configmapsa.yaml

```diff
@@ -1,9 +1,9 @@
-{{- if and (eq .Values.persistentType "sharedAll" ) (eq .Values.omType "cache" ) }}
+{{- if and (eq .Values.bsType "store" ) (eq .Values.omType "cache" ) (eq .Values.storeType "RDBMS" ) }}
```

Comment:
@LakshmiMekala valid values for storeType would be - _**None / Oracle / SQL Server / DB2 / MySQL / PostgreSQL / ActiveSpaces 4x (AS4) / Apache Cassandra**_ but not _**RDBMS**_. 
Please amend the logic accordingly.

Reply:
All the RDBMS related values ( Oracle / SQL Server / DB2 / MySQL / PostgreSQL) will be considered as RDBMS. For details please refer to helm Readme.md file